Gemfile

```diff
@@ -25,6 +25,7 @@ gem 'tilt', '1.4.1'
 gem 'uglifier'
 gem 'uk_postcode', '~> 1.0.1'
 gem 'unicorn', '4.8.3'
+gem 'hashugar'
```

Comment:
In case it helps, I'm pretty sure the deep OpenStruct thing happens in [`GdsApi::Response#to_ostruct`][1] -> [`GdsApi::Response#build_ostruct_recursively `][2].

[1]: https://github.com/alphagov/gds-api-adapters/blob/master/lib/gds_api/response.rb#L77-L79
[2]: https://github.com/alphagov/gds-api-adapters/blob/master/lib/gds_api/response.rb#L120-L129